### ISSUE TITLE
No weather icons shown

### ISSUE THREAD

**Issue by conbart** [librehat/com.librehat.yahooweather#71]
Hi,

I installed the yahooweather plasmoid from your git repository, and build it myself with cmake but no weather icons are shown.
Before that I tried to install it systemwide from the AUR Repository (I'm using manjaro) and via the plasma 5 Desktop to ~/.local/share. With all three installation methods I got the same results: weather infos are shown but no weather icons.
I tried to understand the source code where the icons come from, since there are no image files in the repository but I gave up.
I'm using WOEID 700186 and 24h format. What am I doing wrong?

Greetz,
conbart

**Comment by conbart** [comment]
I figured it out myself: The icons are coming from the plasma 5 icon theme and the manjaro maia theme (which I use) doesn't provide those icons.
I copied the maia theme to ~/.local/share/icons/maia copied the weather icons from the oxygen-icons-svg package to the subfolder status/weather I created and added that folder to the index.theme file. Then updated the cache with gtk-update-icon-cache tool. After a reboot the icons are shown.
